### PR TITLE
Updated npm dependencies

### PR DESCRIPTION
**Pull request by mkArtakMSFT**
Addresses https://github.com/aspnet/AspNetCore/issues/13533

### CODE REVIEW

File: src/ProjectTemplates/Web.Spa.ProjectTemplates/content/React-CSharp/ClientApp/package-lock.json [hunk 1]
```diff
@@ -4868,9 +4868,12 @@
       }
     },
     "eslint-utils": {
-      "version": "1.3.1",
-      "resolved": "https://registry.npmjs.org/eslint-utils/-/eslint-utils-1.3.1.tgz",
-      "integrity": "sha512-Z7YjnIldX+2XMcjr7ZkgEsOj/bREONV60qYeB/bjMAqqqZ4zxKyWX+BOUkdmRmA9riiIPVvo5x86m5elviOk0Q=="
+      "version": "1.4.2",
+      "resolved": "https://registry.npmjs.org/eslint-utils/-/eslint-utils-1.4.2.tgz",
+      "integrity": "sha512-eAZS2sEUMlIeCjBeubdj45dmBHQwPHWyBcT1VSYB7o9x9WRRqKxyUoiXlRjyAwzN7YEzHJlYg0NmzDRWx6GP4Q==",
+      "requires": {
```
Comment (ajaybhargavb) [inline src/ProjectTemplates/Web.Spa.ProjectTemplates/content/React-CSharp/ClientApp/package-lock.json]: I don't have enough context but is this necessary? Looks like it wasn't there before.

**Review by ajaybhargavb — APPROVED** [review]

File: src/ProjectTemplates/Web.Spa.ProjectTemplates/content/React-CSharp/ClientApp/package-lock.json (hunk 1 above)
Comment (mkArtakMSFT) [reply src/ProjectTemplates/Web.Spa.ProjectTemplates/content/React-CSharp/ClientApp/package-lock.json]: These changes are auto-generated by nmp.

**Comment by mkArtakMSFT** [comment]
@aspnet-hello triage https://dev.azure.com/dnceng/public/_build/results?buildId=372349

**Comment by aspnet-hello** [comment]
This comment was made automatically. If there is a problem contact aspnetcore-build@microsoft.com.

I've triaged the above build. I've created/commented on the following issue(s)
https://github.com/aspnet/AspNetCore-Internal/issues/3163
https://github.com/aspnet/AspNetCore-Internal/issues/2996

**Comment by ryanbrandenburg** [comment]
@mkArtakMSFT this will only update the transient dependencies that use floating versions. Is that intended or were you trying to update for a specific reason? Not a lot of context in the linked issue.

**Comment by mkArtakMSFT** [comment]
Just to avoid using vulnerable dependencies

**Pull request merged**